extensions/amp-analytics/0.1/visibility-manager.js

```diff
@@ -702,9 +701,15 @@ export class VisibilityManagerForEmbed extends VisibilityManager {
     return this.backgroundedAtStart_;
   }
 
-  /** @override */
+  /**
+   * Gets the layout box of the embedded document. Note that this may be
+   * smaller than the size allocated by the host. In that case, the document
+   * will be centered, and the unfilled space will not be reflected in this
+   * return value.
+   * @override
+   */
   getRootLayoutBox() {
-    const rootElement = dev().assertElement(this.embed.host);
+    const rootElement = dev().assertElement(this.embed.iframe);
```

Comment:
Can we have a test coverage for this. that the iframe doesn't fill the host element.

Reply:
SG, will do.

Reply:
On second thought, is this sufficient?
https://github.com/ampproject/amphtml/pull/14217/files#diff-962aa285931b08553128aff41fe3da92L786
If it calls with embed.iframe, the test will pass, but if it calls with embed.host, the stub will return null causing the unit test to fail.

Reply:
Ah. good point! I think that's sufficient : )